### PR TITLE
Logging enhancement

### PR DESCRIPTION
**Pull request by triccyx**
Added the `<verbose>` global parameter in `<plugin>`. The parameter is used to have more verbose logging.

### CODE REVIEW

File: plugins/actuator/README.md [hunk 1]
```diff
@@ -22,6 +23,13 @@ Example configuration:
 ~~~
 The plugin filename is `libFMIActuatorPlugin.so` .
 
+### Documentation of the parameters of the `<plugin>` tag.
+| Parameter name | Type    | Description                 | Required  |  Notes |
+|:--------------:|:-------:|:--------------------------: |:---------:|:-----:|
+| verbose        | boolean | Verbose logging setting. | No | Default value is false.
+| actuator       | actuator| Actuators managed by this plugin instance. | Yes | Look at `<actuator>` tag below.|
```
Comment (traversaro) [inline plugins/actuator/README.md]:
```suggestion
| actuator       | actuator | Actuators managed by this plugin instance. | Yes | Look at `<actuator>` tag documentation in the next section. |
```

File: plugins/actuator/README.md [hunk 2]
```diff
@@ -22,6 +23,13 @@ Example configuration:
 ~~~
 The plugin filename is `libFMIActuatorPlugin.so` .
 
+### Documentation of the parameters of the `<plugin>` tag.
+| Parameter name | Type    | Description                 | Required  |  Notes |
+|:--------------:|:-------:|:--------------------------: |:---------:|:-----:|
+| verbose        | boolean | Verbose logging setting. | No | Default value is false.
```
Comment (traversaro) [inline plugins/actuator/README.md]:
```suggestion
| verbose        | boolean | If true, print non-error messages related to plugin. | No | Default value is false. | 
```

File: plugins/actuator/FMIActuatorPlugin.cc [hunk 3]
```diff
@@ -209,8 +214,14 @@ bool FMIActuatorPlugin::ParseParameters(gazebo::physics::ModelPtr _parent, sdf::
         gzwarn << "FMIActuatorPlugin: option jointTorqueName deprecated, please use variable_names tag instead." << std::endl;
         actuator.m_outputVariablesNames[FMIActuatorPluginNS::jointTorque] = elem->Get<std::string>("jointTorqueName");
       }
-
+    
+      if(verbose)
+      {
+        gzwarn << "FMIActuatorPlugin: actuator loaded for joint name:" << jointName << std::endl;
```
Comment (traversaro) [inline plugins/actuator/FMIActuatorPlugin.cc]: Perhaps `gzinfo` is more appropriate? If the plugin is loaded, I do not think we should consider it a warning. 
```suggestion
        gzinfo << "FMIActuatorPlugin: actuator loaded for joint name:" << jointName << std::endl;
```

Comment (triccyx) [reply plugins/actuator/FMIActuatorPlugin.cc]: Ok but I was not able to see on the log if I use gzinfo. Is it due to gzsever log severity?

File: plugins/actuator/README.md [hunk 4]
```diff
@@ -49,6 +57,7 @@ for example if the actuator input variable in your FMU is called `motorInput` an
   ...
   <plugin name="fmi_actuator_plugin" filename="libFMIActuatorPlugin.so">
     <actuator>
+      <verbose>false</verbose>
```
Comment (traversaro) [inline plugins/actuator/README.md]: I am wrong or the `<verbose>` tag should be a child of `<plugin>`, not a child of `<actuator>`?

Comment (triccyx) [reply plugins/actuator/README.md]: Yes you are right.

**Comment by traversaro** [comment]
Can you check the Travis failures? I guess the correct form for the information Gazebo logger is `gzlog`, not `gzinfo` as I incorrectly suggested.

**Comment by triccyx** [comment]
I would use gzmsg as it logs on consol (apparently)

```
/// \brief Output a message
#define gzmsg (gazebo::common::Console::msg())
/// \brief Output a debug message
#define gzdbg (gazebo::common::Console::dbg(__FILE__, __LINE__))
...

/// \brief Output a message to a log file
#define gzlog (gazebo::common::Console::log())
```

**Comment by traversaro** [comment]
Ok!

**Pull request merged**